source/index.txt

```diff
@@ -31,6 +31,11 @@ MongoDB driver for synchronous Java applications. Download it using
 `Gradle <https://gradle.org/>`__, or set up a runnable project by
 following our Quick Start guide.
 
+This documentation is for the synchronous version of the driver. There is also
+an asynchronous version of the driver available. The :driver:`MongoDB Java
+Drivers </java-drivers/>` page lists the available versions of the driver for
```

Comment:
Issue 1: 
I think "versions of the driver" is not a precise term to describe the difference between Java (sync) and Reactive Streams. Also, I think "versions of the driver" could be mistaken for version numbers since we use that language to refer to version numbers in the current documentation (e.g. the left nav item "Upgrade Driver Versions"). 

Suggestion 1:
Avoid using "version" to refer to the different drivers. 


Issue 2: 
I'm not sure it's helpful to link back to java-drivers/. Sorry if my proposed solution on JIRA may have seemed like I was proposing adding it. I was referring to how readers that first landed on the java-drivers/ page wouldn't get anything new from this ticket, but those that landed directly on the wrong driver would benefit.
There's no new information on the java-drivers/ page, so I don't think adding this link is a valuable page visit for the reader.

Suggestion 2: 
Link to the reactive streams driver as suggested my the proposed solution on JIRA.